src/m365/spo/commands/site/site-classic-set.ts

```diff
@@ -191,6 +193,30 @@ class SpoSiteClassicSetCommand extends SpoCommand {
             });
         });
       })
+      .then((): Promise<FormDigestInfo> => {
+        return this.ensureFormDigest(this.spoAdminUrl as string, logger, this.context, this.debug);
+      })
+      .then((res: FormDigestInfo): Promise<void> => {
+        this.context = res;
+        
+        if (!args.options.description) {
+          return Promise.resolve(undefined as any);
+        }
+
+        const requestOptions: any = {
+          url: `${args.options.url}/_api/web`,
+          headers: {
+            'IF-MATCH': '*',
+            'Accept': 'application/json;odata=verbose',
```

Comment:
Since we're not using the metadata, there is no need to set it to `verbose` and we should use `nometadata` instead to avoid sending unnecessary data over the wire